src/ssl.c

```diff
@@ -21564,6 +21583,29 @@ int wolfSSL_RAND_bytes(unsigned char* buf, int num)
     return ret;
 }
 
+
+int wolfSSL_RAND_poll()
+{
+    WOLFSSL_ENTER("wolfSSL_RAND_poll");
```

Comment:
Please move this WOLFSSL_ENTER() below variable declarations.

Reply:
I moved to below variable declaration.